extensionsGallery.json

```diff
@@ -5351,11 +5351,11 @@
 							"files": [
 								{
 									"assetType": "Microsoft.SQLOps.DownloadPage",
-									"source": " https://builds.pieces.app/stages/staging/plugin_vscode/ads-vsix/download"
+									"source": "https://builds.pieces.app/stages/staging/plugin_vscode/ads-vsix/download"
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
-									"source": "https://raw.githubusercontent.com/pieces-app/documentation/main/docs/assets/_azuredatastudio_out/pieces_blk_icon.svg"
+									"source": "https://raw.githubusercontent.com/pieces-app/documentation/main/docs/assets/pfd_logo.png"
```

Comment:
While you're here actually....one more minor thing - could you update this to a commit/tag instead of main? We've had problems before with people deleting stuff and breaking the gallery, and so pinning it to a known version will reduce the chance of that happening here.

(similar what I did here for another extension https://github.com/microsoft/azuredatastudio/pull/25164)